content/blog/2017/06/2017-06-26-share-jenkins-world-keynote-stage.adoc

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Come Share the Jenkins World Keynote Stage with Me!"
+tags:
+- event
+- jenkinsworld
+author: Kohsuke Kawaguchi
+---
+
+link:https://www.cloudbees.com/jenkinsworld[Jenkins World] is approaching fast,
+and we are all busy preparing for the event.
+I've decided to do something different this year as part of my keynote:
+I want to invite a few Jenkins users like you come up on stage with me.
+
+There have been amazing developments in Jenkins over the past year.
+For my keynote, I want highlight how the new Jenkins
+(Jenkinsfile in repo, no more creating jobs, Blue Ocean UI)
```

Comment:
s/Jenkinsfile in repo/Pipeline as code with the `Jenkinsfile`/

s/Blue Ocean UI/Blue Ocean/ and would you mind linking it to /projects/blueocean